students/Gregg/lesson2/grid.py

```diff
@@ -0,0 +1,78 @@
+
+def grid_no_inputs():
+    """Produce a 2x2 grid with sides of length 4"""
+    corner = "+ "
+    h_side = "- "*4
+    h_spacer = " "*8
+    v_side = "| "
+    edge_row = corner+h_side+corner+h_side+corner
+    spacer_row = v_side + h_spacer + v_side +h_spacer + v_side
```

Comment:
Noticing you have syntax differences = no spaces surrounding plus sign in edge_row, but spaces in spacer_row